src/libraries/Microsoft.PowerFx.Json/Public/Types/UserProvidedTypeDefinitions/FormulaTypeSerializerSettings.cs

```diff
@@ -15,15 +15,24 @@ public class FormulaTypeSerializerSettings
         /// </summary>
         public readonly Func<string, RecordType> LogicalNameToRecordType;
```

Comment:
>readonly [](http://example.com/codeflow?start=15&length=8)

nit: we generally need to avoid public *fields*. 
Can you make a property: 
public {get; private set;}